core/renderers/zelos/drawer.js

```diff
@@ -45,3 +45,63 @@ Blockly.zelos.Drawer = function(block, info) {
 };
 goog.inherits(Blockly.zelos.Drawer, Blockly.blockRendering.Drawer);
 
+
+/**
+ * Add steps for the top corner of the block, taking into account
+ * details such as hats and rounded corners.
+ * @protected
+ */
+Blockly.zelos.Drawer.prototype.drawTop_ = function() {
```

Comment:
What's the reason to put this stuff in the zelos drawer instead of in the common drawer?

Reply:
I was avoiding adding functionality in base that was only relevant to zelos. Ideally I think the base adds corners on either side, and thrasos / geras can have right square corners. 
I tried to do that, but I broke the highlight logic.
If you'd prefer I continue down that path, I can refactor this such that the base renderer supports corners on all sides, and deal with the issues that arise in the other renderers.

Reply:
Gotcha.  You can leave it this way, and I'll file a bug to follow up and unify it.